spec/intl.html

```diff
@@ -0,0 +1,584 @@
+              <th>Pattern</th>
+              <th>Fields</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th>[[CivilDatePattern]]</td>
+              <td>[[weekday]], [[era]], [[year]], [[month]], [[day]]</td>
+            </tr>
+            <tr>
+              <th>[[CivilTimePattern]]</td>
+              <td>[[hour]], [[minute]], [[second]]</td>
+            </tr>
+            <tr>
+              <th>[[CivilDateTimePattern]]</td>
+              <td>[[weekday]], [[era]], [[year]], [[month]], [[day]], [[hour]], [[minute]], [[second]]</td>
+            </tr>
+            <tr>
+              <th>[[ZonedDateTime]]</td>
+              <td>[[weekday]], [[era]], [[year]], [[month]], [[day]], [[hour]], [[minute]], [[second]]</td>
+            </tr>
+          </tbody>
+        </table>
+      </emu-table>
+      </ins>
+
+    </emu-clause>
+
+    <emu-clause id="sec-datetime-format-functions">
+      <h1>DateTime Format Functions</h1>
+
+      <p>A DateTime format function is an anonymous built-in function that has a [[DateTimeFormat]] internal slot.</p>
+      <p>When a DateTime format function _F_ is called with optional argument _date_, the following steps are taken:</p>
+
+      <emu-alg>
+        1. Let _dtf_ be _F_.[[DateTimeFormat]].
+        1. Assert: Type(_dtf_) is Object and _dtf_ has an [[InitializedDateTimeFormat]] internal slot.
+        1. <del>If _date_ is not provided or is *undefined*, then</del>
+          1. <del>Let _x_ be Call(%Date_now%, *undefined*).</del>
+        1. <del>Else,</del>
+          1. <del>Let _x_ be ? ToNumber(_date_).</del>
+        1. Return FormatDateTime(_dtf_, <del>_x_</del><ins>_date_</ins>).
+      </emu-alg>
+
+      <p>
+        The `length` property of a DateTime format function is 1.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-partitiondatetimepattern" aoid="PartitionDateTimePattern">
+      <h1>PartitionDateTimePattern ( _dateTimeFormat_, <del>_x_</del><ins>_date_</del> )</h1>
+
+      <p>
+        The PartitionDateTimePattern abstract operation is called with arguments _dateTimeFormat_ (which must be an object initialized as a DateTimeFormat) and <del>_x_</del><ins>_date_</del> (which must be <del>a Number</del><ins>an ECMAScript</ins> value), <del>interprets _x_ as a time value as specified in ES2015, <emu-xref href="#sec-time-values-and-time-range"></emu-xref>,</del> and creates the corresponding parts according to the effective locale and the formatting options of _dateTimeFormat_. The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. <ins>If IsTemporalObject(_date_) is *true*,</ins>
+          1. <ins>Let _x_ be _date_.</ins>
+          1. <ins>If _date_ is a CivilDate object, then</ins>
+            1. <ins>Let _pattern_ be _dateTimeFormat_.[[CivilDatePattern]].</ins>
+            1. <ins>Let _tm_ be { [[weekday]]: _date_.dayOfWeek, [[year]]: _date_.year, [[month]]: _date_.month, [[day]]: _date_.day }.</ins>
```

Comment:
Where do you see `dayOfWeek` in the spec for the Temporal objects?  Or is this just temporary?

For the other fields (year, month, and day), should we access them from an internal slot instead?

Reply:
See reply to the PR thread for more information.